js/ui/gantt/ui.gantt.js

```diff
@@ -304,6 +328,12 @@ class Gantt extends Widget {
             case "treeListWidth":
                 this._updateWidth(args.value);
                 break;
+            case "showResources":
+                this._ganttView && this._ganttView.option("showResources", args.value);
```

Comment:
Perhaps we can define a separate method that will change the GanttView options if view exists (to avoid duplication of `this._ganttView && this._ganttView.option(<...>)`).